velox/docs/functions/datetime.rst

```diff
@@ -37,6 +37,10 @@ The above examples use the timestamp ``2001-08-22 03:04:05.321`` as the input.
 
     Returns ``timestamp`` truncated to ``unit``.
 
+.. function:: date_trunc(unit, date) -> date
```

Comment:
This is a duplicate of the above. Perhaps, edit the existing documentation to clarify that it supports both TIMESTAMP and DATE inputs.

```
.. function:: date_trunc(unit, x) -> x

    Returns ``x`` truncated to ``unit``. The supported types for ``x`` are: TIMESTAMP and DATE.
```